### DISCUSSION TITLE
Support for C++?

### DISCUSSION THREAD

**Discussion by yiyang-msft** [microsoft/Recognizers-Text#2725] [Q&A] [answered]
any recommendation for how to use this library using C++?

**Comment by ahmed0saber** [comment]
which library ?

**Comment by tellarin** [comment, marked as answer]
> any recommendation for how to use this library using C++?

It would depend on your particular requirements. Do you need a OS-specific solution or cross-platform? Can you re-build from the latest code or need to use a released package? Do you need in-place use or can you make a call to a wrapper service/process?

In any case, I'd suggest using the .NET version of the library as it's the most mature and with better quality results.

**Reply by tellarin** [reply]
https://docs.microsoft.com/en-us/dotnet/core/tutorials/netcore-hosting
